components/hdw-tft/hdw-tft.c

```diff
@@ -268,6 +269,9 @@ void initTFT(spi_host_device_t spiHost, gpio_num_t sclk, gpio_num_t mosi, gpio_n
     {
         pixels = (paletteColor_t*)malloc(sizeof(paletteColor_t) * TFT_HEIGHT * TFT_WIDTH);
     }
+    pFrameBuffer = pixels;
+    void SetCachedDPXShapes( uint32_t );
+    SetCachedDPXShapes( (uint32_t)pixels );
```

Comment:
Replace with a call to `getPxTftFramebuffer()` from a new `initShapes()` (or similar)